synfig-core/src/synfig/timepointcollect.cpp

```diff
@@ -58,7 +58,8 @@ using namespace synfig;
 int
 synfig::waypoint_collect(std::set<Waypoint, std::less<UniqueID> >	&waypoint_set,
 						 const Time								&time,
-						 const etl::handle<Node>				&node)
+						 const etl::handle<Node>				&node,
+						 bool initial_waypoint)
```

Comment:
I suggest to change the parameter name to `ignore_dynamic_parameters`.

Reply:
ok, makes sense will do.